prophyc/model.py

```diff
@@ -42,195 +47,323 @@ class Kind:
 """ Enum byte size. """
 ENUM_SIZE = BUILTIN_SIZES['u32']
 
-""" Model consists of 6 kinds of symbols:
-Includes, Constants, Enums, Typedefs, Structs, Unions. """
+model_repr = renew.reproducible(namespace="model")
```

Comment:
Namespace `model` requires user to `from prophyc import model` before repr can be evaluated. If namespace was `prophyc.model`, this could be avoided (although whole expression would be much longer).